.github/workflows/ci.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          pip install -r requirements.txt
+          pip install .[cli]
```

Comment:
This will actually install pyanura + dependencies which seems circular since this CI job is actually meant to package pyanura.